site/src/Layers.js

```diff
@@ -75,59 +85,70 @@ export const layers = [
     point: true,
     line: true,
     activeOnly: true,
-    activeColor: "#8a70ff",
-    color: "#c5b8ff",
+    activeColor: "hsla(210, 15%, 65%, 0.25)",
+    fillOutlineColor: "hsla(209, 13%, 52%, 0.5)",
+    color: "hsla(209, 15%, 58%, 1)",
+    labelColor:"hsla(207, 17%, 37%, 1)",
   },
   {
     theme: "transportation",
     type: "segment",
     line: true,
-    color: "#DECECA",
-    activeColor: "#fa1937",
+    color: "hsla(210, 16%, 82%, 0.4)",
+    activeColor: "hsla(209, 100%, 72%, 0.5)",
+    labelColor:"hsla(214, 100%, 40%, 1)",
   },
   {
     theme: "transportation",
     type: "connector",
     point: true,
-    color: "#DECECA",
-    activeColor: "#fa1937",
+    color: "hsla(210, 16%, 82%, 0.5)",
+    activeColor: "hsla(209, 100%, 72%, 0.5)",
+    labelColor:"hsla(214, 100%, 40%, 1)",
     pointSize: 5,
   },
   {
     theme: "buildings",
     type: "building",
     extrusion: true,
-    color: "#C0C3D8",
-    activeColor: "#4359fe",
+    color: "hsla(210, 16%, 82%, 0.2)",
+    activeColor: "hsla(344, 100%, 66%, 0.4)",
+    labelColor: "hsla(341, 91%, 32%, 1)",
   },
   {
     theme: "buildings",
     type: "building_part",
     extrusion: true,
-    color: "#C0C3D8",
-    activeColor: "#4359fe",
+    color: "hsla(210, 16%, 82%, 0.2)",
+    activeColor: "hsla(344, 100%, 66%, 0.4)",
+    labelColor: "hsla(341, 91%, 32%, 1)",
   },
   {
     theme: "places",
     type: "place",
     point: true,
-    color: "#fdcd96",
-    activeColor: "#fb8804",
+    minzoom: 11,
+    color: "hsla(226, 100%, 75%, 1)",
+    activeColor: "hsla(243, 100%, 61%, 0.25)",
+    labelColor: "hsla(249, 83%, 39%, 1)",
+    pointSize: 5,
   },
   {
     theme: "addresses",
     type: "address",
     point: true,
-    color: "#ffadf1",
-    activeColor: "#ff1ad9",
+    color: "hsla(210, 15%, 65%, 1)",
+    activeColor: "hsla(43, 100%, 42%, 0.5)",
+    labelColor: "hsla(43, 100%, 42%, 1)",
     pointSize: 5,
   },
   {
     theme: "divisions",
-    type: "boundary",
+    type: "division_boundary",
```

Comment:
Changing the type here is absolutely fine, but it has made me realize that our component doesn't render well with a long label: 

Division
Boundary 

is too long and so it wraps, making the formatting look strange. We can tackle this in a follow on, or we can just change both types from 'division area/division boundary' to just 'area/boundary'. I feel like the latter is okay since.... it should be obvious these already fall under divisions. 

<img width="169" alt="image" src="https://github.com/user-attachments/assets/9e91e172-f10b-45e9-880c-d0da9895d6e8">